### ISSUE TITLE
asp-append-version does not work for RCL files

### ISSUE THREAD

**Issue by datvm** [aspnet/AspNetCore#13047]
### Describe the bug
`asp-append-version` does not work for files embedded by Razor Class Library when referred to by `script`, `link` or `img` tags, either in the same RCL or by the client (using) project.

### To Reproduce
1. Create a RCL project (I am using 3.0 preview 7).

2. Create wwwroot folder, and create a file, for example `css\common.min.css`.

3. Create a Razor page (Layout or Partial, does not matter) and add a reference to the file:

`<link async rel="stylesheet" href="~/css/common.min.css" asp-append-version="true" />`

4. Create a Razor Page project and refer to the RCL project. The output does not have `?v=hash` attached:

`<link async rel="stylesheet" href="/css/common.min.css" />`

### Expected behavior
The output `href` should be `/css/common.min.css?v=some-hash`.

### Screenshots
The output is without the `?v=hash`:

![image](https://user-images.githubusercontent.com/6388546/62858581-e8940d80-bd24-11e9-8efa-e3f2172a3363.png)

### Additional context
I am using [this](https://stackoverflow.com/questions/53518244/is-there-a-built-in-way-to-get-a-string-of-file-path-with-version-like-asp-appen) as a workaround right now.

**Comment by mkArtakMSFT** [comment]
Thanks for contacting us, @datvm.
@javiercn can you please look into this? Thanks.

**Comment by mkArtakMSFT** [comment]
@javiercn, let's first understand what this will take to fix before making any changes.

**Comment by javiercn** [comment]
It might be something here. https://github.com/aspnet/AspNetCore/blob/master/src/Hosting/Hosting/src/StaticWebAssets/StaticWebAssetsFileProvider.cs#L67-L70

**Comment by datvm** [comment]
Hi, sorry, this is actually my mistake.

> Consume content from a referenced RCL
The files included in the wwwroot folder of the RCL are exposed to the consuming app under the prefix _content/{LIBRARY NAME}/. For example, a library named Razor.Class.Lib results in a path to static content at _content/Razor.Class.Lib/.

I used the wrong path for consuming the files (and received 404 error too). I didn't know I still need to use _content/ even when the Razor Page is in the same RCL. I fixed the path and the `asp-append-version` works correctly.

Sorry again.
